source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+********************************************
+Enterprise Mobile Management (EMM) Providers
+********************************************
+
+EMM Providers develop software that helps enterprise teams manage secure mobile technology deployments. This includes the use of mobile devices and usage ready applications.
+
+Most large enterprise teams are familiar with `Enterprise Mobile Management <https://en.wikipedia.org/wiki/Enterprise_mobility_management>`_ providers, or EMMs. If this is a new term for you, review `Appendix B <https://docs.google.com/document/d/1BhHla4medUGVSWuBPNuoz_h5f85MdlIQWcRn5N2WUos/edit#heading=h.pn9oiqr3rebj>`_ for a list of providers and relevant information.
+
+For those taking on larger deployments, we assume you are already using an EMM Provider.
+
+The AppConfig standard is the critical element when deploying Mattermost via an EMM.
+
+AppConfig is newer, more modern approach when compared to App Wrapping.
+
+We’ll say it several times throughout this guide, and in the next section. If you are using an EMM provider, the only supported path is via AppConfig.
+
+While “App Wrapping” has been widely supported, **it is not recommended**. Further, **it is not a supported path** for Mattermost mobile deployment.
+
+AppConfig (Supported) vs. App Wrapping (Not Supported)
+======================================================
+
+Mattermost only supports the AppConfig standard. It does not support app wrapping. If you proceed with app wrapping, it will lead to an inability to use the mobile app.
+
+So App Wrapping isn’t supported, and the AppConfig standard is, but what is the difference between the two?
+
+While there are `some great articles <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_ defining both, let’s take a look at AppConfig first.
+
+AppConfig
+---------
+
+“A community focused on providing tools and best practices around native capabilities in mobile operating systems to enable a more consistent, open and simple way to configure and secure mobile apps in order to increase mobile adoption in business.” - AppConfig.org
+
+So that sounds great, but are there any benefits for users?
+
+Again, in the AppConfig Community’s words, “Users benefit with instant mobile productivity and a seamless out-of-the box experience, and businesses benefit with secure work-ready apps with minimal setup required while leveraging existing investments in Enterprise Mobility Management (EMM), VPN, and identity solutions. Put another way, your apps are simpler to configure, secure and deploy.”
+
+For now, focus on that last part, “... your apps are simpler to configure, secure and deploy.” AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
+
+Application (App) Wrapping
```

Comment:
Also known as `App Containerization`

Reply:
I’m not against that terminology, but it did not come up in any interviews/conversation, and our own documentation uses “App Wrapping”. Aligning for the sake of consistency is best. “ App wrapping via an EMM provider is untested, and not supported by Mattermost. Instead, we recommend using an EMM provider that supports Managed App Configuration.” https://docs.mattermost.com/mobile/mobile-overview.html